recipes/rsatoolbox/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "rsatoolbox" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rsatoolbox-{{ version }}.tar.gz
+  sha256: 2d091cbaa33373bf9da4df5ca8d127f0e427431a3db726076090ab2d54fe1213
+
+build:
+  skip: true  # [py<37 or py>310 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - cython # >=3.0.0a11
+    - numpy >=1.21.2
```

Comment:
You don't want this constraint here or if you do, you need to specify the minimum version. Otherwise, conda will install the latest version of numpy at build time (e.g., 1.24 or w/e) and you won't be able to run against say 1.23.

Reply:
I'm not sure I understand fully, I thought `>=1.21.2` would mean that the minimum is `1.21.2`? Either way I've let go of the version constraints as this seems to be the typical way of doing things in conda, and it seems unlikely that people would now have something old like `1.19`.